mw-webapp/src/component/userCard/UserCard.tsx

```diff
@@ -109,15 +109,25 @@ export const UserCard = observer((props: UserCardProps) => {
         </VerticalContainer>
 
         <VerticalContainer className={styles.additionalInfo}>
-          <p>
-            {`${LanguageService.allUsers.userCard.createdAt[language]}
-            ${DateUtils.getShortISODateValue(props.userPreview.createdAt)}`}
-          </p>
-          <p>
-            {`${LanguageService.allUsers.userCard.ownWays[language]} (${props.userPreview.ownWays}) 
-          ${LanguageService.allUsers.userCard.favoriteWays[language]} (${props.userPreview.favoriteWays})
-          ${LanguageService.allUsers.userCard.mentoringWays[language]} (${props.userPreview.mentoringWays})`}
-          </p>
+          <HorizontalContainer>
+            <span>
+              {`${LanguageService.allUsers.userCard.createdAt[language]}`}
+            </span>
+            <span className={styles.createdAt}>
+              {DateUtils.getShortISODotSplitted(props.userPreview.createdAt)}
+            </span>
+          </HorizontalContainer>
+          <HorizontalContainer className={styles.ways}>
+            <span>
+              {`${LanguageService.allUsers.userCard.ownWays[language]} (${props.userPreview.ownWays})`}
```

Comment:
Is it possible don't use span wrappers for each element?

Reply:
If I remove the wrappers, there will be a small distance between "ways", not the same as in the design.

Design:
![image](https://github.com/tritonJS826/masters-way/assets/8752900/bc54b523-2cc0-4aff-a4af-749d1fc6f0cc)

Without wrappers:
![image](https://github.com/tritonJS826/masters-way/assets/8752900/c2386916-5d30-4629-8566-11cb0b37f3cc)